chain/chain/src/resharding/manager.rs

```diff
@@ -138,9 +135,8 @@ impl ReshardingManager {
         self.process_memtrie_resharding_storage_update(
             chain_store_update,
             block,
-            shard_uid,
```

Comment:
Not you PR
It's a bit worrying that the shard uid is stored in two places and that it's not clearly indicated whether it's parent_shard_uid or child_shard_uid. Maybe a good first issue?

Reply:
I guess it's fine if it's dynamically checked.